linters/.markdownlint.json

```diff
@@ -0,0 +1,33 @@
+{
+  "comment": "Enable all rules, and then disable selective ones https://github.com/DavidAnson/markdownlint/blob/master/doc/Rules.md",
+  "default": true,
+
+  "line-length": false,
+  "ul-start-left": false,
+  "no-multiple-blanks": false,
+  "no-emphasis-as-header": false,
+
+  "comment": "Needed for explicit anchors",
+  "no-inline-html": false,
+
+  "comment": "Rules use '-', some other lists use '+'",
+  "ul-style": false,
+
+  "comment": "LICENCE.md does not conform",
```

Comment:
That tells github how to display it; it's valid both with and without the extension.